storagenode/contact/chore.go

```diff
@@ -48,53 +58,108 @@ func (chore *Chore) Run(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
 	chore.log.Info("Storagenode contact chore starting up")
 
+	var group errgroup.Group
+
 	if !chore.service.initialized.Wait(ctx) {
 		return ctx.Err()
 	}
+	chore.mu.Lock()
+	for _, satellite := range chore.trust.GetSatellites(ctx) {
+		satellite := satellite
+		backOff := time.Second
```

Comment:
It looks like a `backOff` variable is initialized for each satellite, but the value never changes. Maybe we could make this a package level constant instead?